servers/rendering/renderer_rd/storage_rd/mesh_storage.cpp

```diff
@@ -1040,6 +1070,12 @@ void MeshStorage::_mesh_surface_generate_version_for_input_mask(Mesh::Surface::V
 	uint32_t attribute_stride = 0;
 	uint32_t skin_stride = 0;
 
+	enum PreviousLocation {
+		PREVIOUS_LOCATION_VERTEX = 12,
+		PREVIOUS_LOCATION_NORMAL = 13,
+		PREVIOUS_LOCATION_TANGENT = 14
+	};
+
```

Comment:
I'm unsure of where exactly would be the best place to define the locations of the inputs of the shader.